ucfs-claimant-api-mysql-interface.tf

```diff
@@ -54,3 +54,21 @@ resource "null_resource" "ucfs_claimant_api_mysql_interface" {
     command = "./initial-commit.sh ${github_repository.ucfs_claimant_api_mysql_interface.name} '${github_repository.ucfs_claimant_api_mysql_interface.description}' ${github_repository.ucfs_claimant_api_mysql_interface.template.0.repository}"
   }
 }
+
+resource "github_actions_secret" "ucfs-claimant-api-mysql-interface_github_email" {
```

Comment:
resources with underscores please

Reply:
Good point, my eyes did not pick this up 😂